docs/install-pmm/install-pmm-server/aws/aws.md

```diff
@@ -1,47 +1,38 @@
-
-    For more information, see our blog post [How much disk space should I allocate for Percona Monitoring and Management?](https://www.percona.com/blog/2017/05/04/how-much-disk-space-should-i-allocate-for-percona-monitoring-and-management/).
-
-To install PMM Server on AWS:
+To install Percona Monitoring and Management (PMM) Server from AWS Marketplace:
 {.power-number}
 
-1. Click **Continue to Subscribe**.
-
-2. **Subscribe to this software**: Check the terms and conditions and click *Continue to Configuration*.
-
-3. **Configure this software**:
+1. Go to [AWS Marketplace](https://aws.amazon.com/marketplace) and search for **Percona Monitoring and Management Server** or [access the PMM Server listing] directly.
 
-    1. Select a value for **Software Version**. (The latest is {{release}}.)
-    2. Select a region. (You can change this in the next step.)
-    3. Click **Continue to Launch**.
+2. Click **Continue to Subscribe** on the PMM Server listing page, review the terms and conditions, then click **Continue to Configuration**.
 
-4. **Launch this software**:
+3. Select the latest version (recommended), choose the AWS region where you want to deploy PMM, then click **Continue to Launch**.
+4. Choose **Launch from Website** to configure and launch directly from the AWS Marketplace or **Launch through EC2** if you prefer launching via the EC2 Management Console for more customization.
+5. In the **EC2 Instance Type** field, select an appropriate instance type based on your monitoring needs and anticipated load. 
+6. In the **VPC Settings** field, choose an existing VPC or create a new one to host your PMM Server.
+7. In the **Subnet Settings** field, select an existing subnet or create a new one within your VPC.
+8. In the **Security Group Settings** field, choose an existing security group or create a new one based on the default settings provided by the seller.
+9. In the **Key Pair Settings** field, select an existing key pair for SSH access, or create a new one if necessary.
+10. Click **Launch** to deploy the PMM Server.
+11. Once the instance is launched, it will appear in the EC2 console.
 
-    1. **Choose Action**: Select a launch option. **Launch from Website** is a quick way to make your instance ready. For more control, choose *Launch through EC2*.
+Make sure to assign a meaningful name to the instance to help distinguish it from others in your environment.
 
-    2. **EC2 Instance Type**: Select an instance type.
+## Security consideration
 
-    3. **VPC Settings**: Choose or create a VPC (virtual private cloud).
+- Ensure that your security group allows inbound traffic on ports **22** (SSH), **80** (HTTP), and **443** (HTTPS).
+- If PMM needs to monitor an RDS instance, make sure it can access port **3306** on the RDS.
```

Comment:
it might be 3306 for MySQL and 5432 for PostgreSQL.
It's not really related to this page, it's more related to RDS page